cypher/cypher-docs/src/docs/dev/ql/administration/security/role-management-syntax.asciidoc

```diff
@@ -40,7 +40,7 @@ CREATE ROLE name [IF NOT EXISTS] [AS COPY OF name]
 ----
 CREATE OR REPLACE ROLE name [AS COPY OF name]
 ----
-| Create a new role.
+| Create a new role, replacing an existing role if one exists with the same name.
```

Comment:
A suggestion for rewording:
"Create a new role, or if a role with the same name exists, replace it."

Reply:
Or even: Create or replace a role with the specified name.

Reply:
I updated this wording once from our code review, from `Recreate a role, or create a new role if no such role exists.` to `Create a new role, replacing an existing role if one exists with the same name.`

Think I prefer your first version
```suggestion
| Create a new role, or if a role with the same name exists, replace it.
```